test/src/api/commands/testSendKeys.js

```diff
@@ -3,7 +3,8 @@ const MockServer  = require('../../../lib/mockserver.js');
 const CommandGlobals = require('../../../lib/globals/commands.js');
 
 describe('sendKeys', function() {
-
+  this.timeout(12000);
```

Comment:
Increased timeout to handle the **undefined** case because I have faced a timeout error after throwing an error from `sendKeysToElement` fucntion.

Reply:
can we change retry interval instead of this change:

` before(function(done) {
    CommandGlobals.beforeEach.call(this, done, {
      output: false,
      globals: {
        waitForConditionPollInterval: 10,
        waitForConditionTimeout: 11
      }
    });
  });`

Reply:
Thanks @gravityvi. It's working. Changes pushed [c5901a1](https://github.com/nightwatchjs/nightwatch/pull/3115/commits/c5901a1c34231ba872788b08ebc2a40e31d57e93)